api/users.js

```diff
@@ -22,11 +22,11 @@ router.get('/info', async (req, res) => {
 			attributes: { exclude: ['password'] },
 			include: [User],
 		})
-		res.json({
-			students: students.map(student => student.toJSON()),
-			teachers: teachers.map(teacher => teacher.toJSON()),
-			administrators: administrators.map(administrator => administrator.toJSON()),
-		})
+		res.json([
+			...students.map(student => student.toJSON()),
+			...teachers.map(teacher => teacher.toJSON()),
+			...administrators.map(administrator => administrator.toJSON())
+		])
```

Comment:
Nechal bych to jako objekt a ne pole polí ;)

Reply:
Ne, takhle to je lépe zpracovatelné na FE a je to mimochodem pole objektů.

Reply:
Dává větší smysl to vracet jako objekt ať se ví kdo je kdo

Reply:
To určitě ano, ale musí se změnit zase FE, okey není problém, předělám.